back-end/src/main/java/com/exadel/team3/backend/services/email/email_types/EmailTypes.java

```diff
@@ -0,0 +1,42 @@
+package com.exadel.team3.backend.services.email.email_types;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public enum EmailTypes {
+
+    CONFIRM_EMAIL("email_sender/text_for_email/Text_for_confirm_email.txt", "email_sender/subject_for_email/Subject_for_congirm_email.txt"),
```

Comment:
Move this part to the constant as long as it is the same for all types:
email_sender/text_for_email/